File: java/loccount/src/de/lieser_online/ConsoleUi.java
package de.lieser_online;

public class ConsoleUi {
    private final LocStat sum = new LocStat();

    public void showLoc(LocStat locstat) {
        sum.Loc += locstat.Loc;
        sum.Total += locstat.Total;
        System.out.println(locstat.Filename + ", " + 
            locstat.Loc + ", " + locstat.Total);
    }

    public void showSum() {
        System.out.println();
        System.out.println("Sum:");
        System.out.println("  LOC:   " + sum.Loc);
        System.out.println("  Total: " + sum.Total);
    }
}

File: python/loccount/consoleui.py
class ConsoleUi:
    sum_total = 0
    sum_loc = 0

    def show_loc(self, locstat):
        filename, total, loc = locstat
        self.sum_total += total
        self.sum_loc += loc
        print(filename + " " + str(total) + " " + str(loc))

    def show_sum(self):
        print("")
        print("Total")
        print("  Lines: " + str(self.sum_total))
        print("  LOC:   " + str(self.sum_loc))
